### PR TITLE
#87 Vaidate CCI URL

### LINKED ISSUE #87: [conan center] check URL points to the https://github.com/conan-io/conan-center-index
check that URL attribute of `conanfile.py` points to the `https://github.com/conan-io/conan-center-index` (not to the bincrafters or conan-community, for instance)

### PR DESCRIPTION
**Pull request by uilianries**
Since CCI is the new central repo, all URLs should be the same.

closes #87 
Wiki:

#### **<a name="KB-H027">#KB-H027</a>: "CONAN CENTER INDEX URL"**

The attribute `url` should point to the address where the recipe is located.
The current Conan Center Index address is https://github.com/conan-io/conan-center-index

### CODE REVIEW

File: hooks/conan-center.py [hunk 1]
```diff
@@ -189,6 +190,13 @@ def test(out):
             out.error("The size of your recipe folder ({} KB) is larger than the maximum allowed"
                       " size ({}KB).".format(total_size_kb, max_folder_size))
 
+    @run_test("KB-H027", output)
+    def test(out):
+        url = getattr(conanfile, "url", None)
+        if url and "https://github.com/conan-io/conan-center-index" not in url:
```
Comment (SSE4) [inline hooks/conan-center.py]: I think `startswith` is better here?

Comment (danimtb) [reply hooks/conan-center.py]: Yes, I also think it is better as someone could include the `.git` ending or a trailing `/`

File: hooks/conan-center.py [hunk 2]
```diff
@@ -25,7 +25,8 @@
              "KB-H018": "LIBTOOL FILES PRESENCE",
              "KB-H019": "CMAKE FILE NOT IN BUILD FOLDERS",
              "KB-H020": "PC-FILES",
-             "KB-H021": "MS RUNTIME FILES"}
+             "KB-H021": "MS RUNTIME FILES",
+             "KB-H027": "CCI URL"}
```
Comment (lasote) [inline hooks/conan-center.py]: Don't use CCI, Write Conan Center Index

Comment (uilianries) [reply hooks/conan-center.py]: done!

File: hooks/conan-center.py (hunk 1 above)
Comment (uilianries) [reply hooks/conan-center.py]: done!

**Comment by uilianries** [comment]
> Write the description of the KB (that will go in the wiki) in the PR comment.

@lasote done: conan-io/conan-center-index/wiki/Error-Knowledge-Base#kb-h027-conan-center-index-url

**Comment by lasote** [comment]
> Write the description of the KB (that will go in the wiki) in the PR comment.

That means, NOT NOW in the wiki, but in this PR body, to move it when we merge it.
:radioactive: is affecting you again? LoL

**Comment by uilianries** [comment]
Sorry, misreading!

facepalm

**Comment by uilianries** [comment]
#### **<a name="KB-H027">#KB-H027</a>: "CONAN CENTER INDEX URL"**

The attribute `url` should point to the address where the recipe is located.
The current Conan Center Index address is https://github.com/conan-io/conan-center-index

**Comment by danimtb** [comment]
Please resolve the conflicts here. Thx

**Comment by danimtb** [comment]
Some jobs already passed in CI. Merging as this is just a recipe syntax check

**Comment by danimtb** [comment]
KB error was added to the wiki here: https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#kb-h027-conan-center-index-url

**Pull request merged**